src/main/java/core/basesyntax/servise/operation/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.servise.operation;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void apply(String fruit, int quantity, Map<String, Integer> storage) {
+        storage.put(fruit, quantity);
+    }
```

Comment:
Make sure to follow the encapsulation principle. The `apply` method should ensure that the quantity cannot be a negative value. If a negative value is passed, it could lead to incorrect behavior of the program.